lib/aoc_rb/app.rb

```diff
@@ -126,6 +130,12 @@ def spec(year = options[:year], day = options[:day])
       Kernel.exec( "bundle exec rspec #{spec_dir}" )
     end
 
+    desc "spec:all", "runs all tests"
+
+    define_method(:'spec:all') do
```

Comment:
For some reason I had it in my head that thor tasks could be namespaced the same way rake tasks can be. Perhaps we should just use an `--all` option instead, as per the standard usage of thor?

Reply:
Yeah it looks a lot better, got it working as you mentioned. Tests are passing now too